clients/privacy-center/constants/index.ts

```diff
@@ -1,6 +1,9 @@
 /* eslint-disable import/prefer-default-export */
 
-import config from "../config/config.json";
+import { Config } from "~/types/config";
+import configJson from "~/config/config.json";
+
+export const config: Config = configJson;
```

Comment:
Note that we still get the benefit of typechecking with this. The imported JSON has a typed structure, so typescript will throw an error here if the JSON isn't assignable to the canonical type.